### ISSUE TITLE
.pem certificate creation in iothub raise cli.azure.cli.core.azclierror: (500019) Internal Error.

### ISSUE THREAD

**Issue by diarraa** [Azure/azure-iot-sdk-python#809]
Hi,

I try to upload a pem certificate in iothub using the following command:
 **az iot hub certificate create --hub-name myIoThub --name MyCertificate --path .\mycert.pem**

It fails by raising this error message:

cli.azure.cli.core.azclierror: (500019) Internal Error. If you contact a support representative please include this correlation identifier: 7004c644-0731-4401-a41e-fac12b3c9fed, timestamp: 2021-07-01 12:36:55Z, errorcode: IH500019.
az_command_data_logger: (500019) Internal Error. If you contact a support representative please include this correlation identifier: 7004c644-0731-4401-a41e-fac12b3c9fed, timestamp: 2021-07-01 12:36:55Z, errorcode: IH500019.

Thanks you in advance
AD

**Comment by digimaun** [comment]
I was not able to repro this issue. These are the steps I took:

Ensure you have a recent CLI version installed. For this test I used `2.17.1`.
![image](https://user-images.githubusercontent.com/28658112/124169957-b0eaf780-da5b-11eb-9689-d9a049c47350.png)

Then ensure you have a well formed CA cert for upload. These are the commands I used to generate a fake CA for this test:

`openssl genrsa -des3 -out myFakeCA.key 2048`
`openssl req -x509 -new -nodes -key myCA.key -sha256 -days 1825 -out myFakeCA.pem`

I then used the same command you did -- this worked:

![image](https://user-images.githubusercontent.com/28658112/124170601-7c2b7000-da5c-11eb-9ebe-331d237388fb.png)

And shows up properly in portal:

![image](https://user-images.githubusercontent.com/28658112/124170714-9e24f280-da5c-11eb-935a-a882c33085c9.png)
